packs/octopusdeploy/actions/create_release.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: create_release
+runner_type: run-python
```

Comment:
Seems `run-python` runner type is outdated naming. 
http://docs.stackstorm.com/actions.html?highlight=runner_type#available-runners
It will work, but better to replace with `python-script` for consistency and less confusion in future.